docs/source/example_notebooks/dowhy_causal_discovery_example.ipynb

```diff
@@ -0,0 +1,692 @@
+    "    'PC' : PC,\n",
+    "    'GES' : GES,\n",
+    "}\n",
+    "\n",
+    "for method, lib in functions.items():\n",
+    "    obj = lib()\n",
+    "    output = obj.predict(data_mpg)\n",
+    "    adj_matrix = nx.to_numpy_matrix(output)\n",
+    "    adj_matrix = np.asarray(adj_matrix)\n",
+    "    graph_dot = make_graph(adj_matrix, labels)\n",
+    "    graphs[method] = graph_dot\n",
+    "\n",
+    "# Visualize graphs\n",
+    "for method, graph in graphs.items():\n",
+    "    print(\"Method : %s\"%(method))\n",
+    "    display(graph)"
+   ]
+  },
+  {
+   "source": [
+    "## Estimate effects using Linear Regression"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "\n",
+      "*****************************************************************************\n",
+      "\n",
+      "Causal Discovery Method : LiNGAM\n",
+      "Estimand type: nonparametric-ate\n",
+      "\n",
+      "### Estimand : 1\n",
+      "Estimand name: backdoor\n",
+      "Estimand expression:\n",
+      "  d                                               \n",
+      "──────(Expectation(weight|acceleration,cylinders))\n",
+      "d[mpg]                                            \n",
+      "Estimand assumption 1, Unconfoundedness: If U→{mpg} and U→weight then P(weight|mpg,acceleration,cylinders,U) = P(weight|mpg,acceleration,cylinders)\n",
+      "\n",
+      "### Estimand : 2\n",
+      "Estimand name: iv\n",
+      "No such variable found!\n",
+      "\n",
+      "### Estimand : 3\n",
+      "Estimand name: frontdoor\n",
+      "No such variable found!\n",
+      "\n",
+      "Causal Estimate is -39.56010817288552\n",
+      "\n",
+      "*****************************************************************************\n",
+      "\n",
+      "Causal Discovery Method : PC\n",
+      "Estimand type: nonparametric-ate\n",
+      "\n",
+      "### Estimand : 1\n",
+      "Estimand name: backdoor\n",
+      "No such variable found!\n",
+      "\n",
+      "### Estimand : 2\n",
+      "Estimand name: iv\n",
+      "No such variable found!\n",
+      "\n",
+      "### Estimand : 3\n",
+      "Estimand name: frontdoor\n",
+      "No such variable found!\n",
+      "\n",
+      "Causal Estimate is None\n",
+      "\n",
+      "*****************************************************************************\n",
+      "\n",
+      "Causal Discovery Method : GES\n",
+      "Estimand type: nonparametric-ate\n",
+      "\n",
+      "### Estimand : 1\n",
+      "Estimand name: backdoor\n",
+      "No such variable found!\n",
+      "\n",
+      "### Estimand : 2\n",
+      "Estimand name: iv\n",
+      "No such variable found!\n",
+      "\n",
+      "### Estimand : 3\n",
+      "Estimand name: frontdoor\n",
+      "No such variable found!\n",
+      "\n",
+      "Causal Estimate is None\n"
```

Comment:
None because the graphs PC and GES output are undirected. You can see that there are no backdoor, frontdoor or IVs. This is what I was telling you about the last day. How do we deal with this?